src/pages/SendTokens/TokenInfoBlock.js

```diff
@@ -0,0 +1,72 @@
+import { Paper, Grid, Typography } from '@mui/material';
+import { styled } from '@mui/system';
+import tokenIcon from './tokens.svg';
+
+
+const SubText = styled(Typography)({
```

Comment:
Could you please move these components to a separate file like TokenInfoBlockStyled.js?